pkg/assembler/instruction.go

```diff
@@ -161,6 +160,9 @@ type Instruction struct {
 
 	// Defines which instruction needs to be executed
 	Opcode Opcode
+
+	// Immediate value for 2 word instructions
+	Imm string
```

Comment:
We don't need an Imm as a field element of an instruction

Reply:
I changed the way we convert AST to instruction. I made an interface Word that keeps either an Instruction or Immediate (alias for string for better readability). so now `astToInstruction` returns a "wordList" and we can use this to encode it into bytecode in `instruction.go`. What do you think?